### ISSUE TITLE
Wrong warning LC0035 for Fields with Access Local/Protected

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#578]
Currenlty LC0035 is triggered event for fields defined with Access Local or Protected.

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/56249171/8b784ca7-c629-4dd8-a2b4-ad069bcf7a2d)

```al
        field(30; "Client Secret Key"; Guid)
        {
            Access = Local;
            Caption = 'Client Secret ID';
            DataClassification = SystemMetaData;
        }
```

This shouldn't be the case, since those fields cannot be added to pages.

**Comment by Arthurvdv** [comment]
Thanks! This indeed a good improvement for this rule and now be resolved in the (pre)release version of [v0.30.16](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.16) of the LinterCop.

**Comment by Arthurvdv** [comment]
Version [v0.30.16](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.16) of the LinterCop is now released. Is it possible to verify of the issue is now resolved?
